### DISCUSSION TITLE
Need a nice looking "Connect with BTCPay Server" button as HTML with SVG icon to copy-paste in our eCommerce backends

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2911] [General]
The eCommerce integrations we're building could use a "Connect with BTCPay Server" button.
For this we need some nice looking HTML+SVG that we can copy-paste in every module/plugin/integration.

@dstrukt could you help us out with this?

The look & feel should be final, as we cannot easily change this once deployed to shopkeepers.
The HTML should have inline CSS and be nice looking by itself.

I can help out with the HTML stuff, but I don't know what's current design-wise...

**Comment by dstrukt** [comment]
Hey - absolutely, will prioritize this for today.
